src/test/skylark/testdata/string_misc.sky

```diff
@@ -62,6 +62,17 @@ assert_eq('Apricot'.endswith('co', -1), False)
 assert_eq('abcd'.endswith('c', -2, -1), True)
 assert_eq('abcd'.endswith('c', 1, 8), False)
 assert_eq('abcd'.endswith('d', 1, 8), True)
+assert_eq('Apricot'.endswith(('cot', 'toc')), True)
+assert_eq('Apricot'.endswith(('toc', 'cot')), True)
+assert_eq('a'.endswith(('', '')), True)
+assert_eq('a'.endswith(('', 'a')), True)
+assert_eq('a'.endswith(('a', 'a')), True)
+assert_eq(''.endswith(('a', '')), True)
+assert_eq(''.endswith(('', '')), True)
+assert_eq(''.endswith(('a', 'a')), False)
+---
+'a'.endswith(['a']) ### expected value of type 'string or tuple of strings' for parameter 'sub', in method call endswith(list) of 'string'
+---
```

Comment:
Can we add cases for:
- singleton tuple
- empty tuple
- tuple with element of wrong type

Reply:
Done

Reply:
Thanks.